src/graphql/root/mutation/ln-invoice-payment-send.ts

```diff
@@ -25,11 +25,11 @@ const LnInvoicePaymentSendMutation = GT.Field({
   args: {
     input: { type: GT.NonNull(LnInvoicePaymentInput) },
   },
-  resolve: async (_, args, { ip, domainUser, logger }) =>
+  resolve: async (_, args, { ip, domainUser, domainAccount, logger }) =>
     addAttributesToCurrentSpanAndPropagate(
       {
         [SemanticAttributes.ENDUSER_ID]: domainUser?.id,
-        [ENDUSER_ALIAS]: domainUser?.username,
+        [ENDUSER_ALIAS]: domainAccount?.username,
```

Comment:
Should probably rename `ENDUSER_ALIAS` to reflect the change in domain structure as well

Reply:
yup make sense. The name `ENDUSER_ALIAS` is not a standard by open telemetry? what did you choose `ENDUSER_ALIAS` rather than `ENDUSER_USERNAME`?